docs/src/data/mockServices.tsx

```diff
@@ -0,0 +1,89 @@
+import { Amplify} from 'aws-amplify';
+
+const awsExports = {
+  aws_user_pools_id: 'xx-xxxx-x_xxxxx', // (optional) -  Amazon Cognito User Pool ID
+  aws_user_pools_web_client_id: 'xxxxxxxxxxxxxx', // (optional) - Amazon Cognito App Client ID (App client secret needs to be disabled)
+};
+Amplify.configure(awsExports)
+
+let signUpPassword = "";
+const verifiedUsers = new Map();
+const fixed_username = "amplify-ui";
+const fixed_password = "connected-components";
```

Comment:
I wonder if we can genericize these values for easier reference. Maybe just username "test" and password "password"?

Reply:
Yes, we need to use an easy one. And for constant, please use the `CONSTANT_CASE`

Reply:
Done!